### PR TITLE
[zh-cn]: create the translation of `VirtualKeyboard.show()` method

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/web/api/virtualkeyboard/show/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: VirtualKeyboard：show() 方法
+slug: Web/API/VirtualKeyboard/show
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+{{domxref("VirtualKeyboard")}} 接口的 **`show()`** 方法用于以编程方式显示屏幕上的虚拟键盘。在页面需要实现自己的虚拟键盘逻辑时特别有用，特别是当使用 `virtualkeyboardpolicy` 属性在 `contenteditable` 元素上，如[控制 `contenteditable` 元素上的虚拟键盘](/zh-CN/docs/Web/API/VirtualKeyboard_API#control_the_virtual_keyboard_on_contenteditable_elements)所解释的那样。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/virtualkeyboard/show/index.md:10]: 另外anchor有问题

```suggestion
{{domxref("VirtualKeyboard")}} 接口的 **`show()`** 方法用于以编程方式显示屏幕上的虚拟键盘。这在页面需要实现自己的虚拟键盘逻辑时特别有用，特别是在 `contenteditable` 元素上使用 `virtualkeyboardpolicy` 属性时，如[控制 `contenteditable` 元素上的虚拟键盘](/zh-CN/docs/Web/API/VirtualKeyboard_API#control_the_virtual_keyboard_on_contenteditable_elements)所解释的那样。
```

File: files/zh-cn/web/api/virtualkeyboard/show/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+show()
+```
+
+### 参数
+
+无。
+
+### 返回值
+
+未定义。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/virtualkeyboard/show/index.md:28]:
```suggestion
Undefined。
```

这里指返回值是一个 undefined 值

File: files/zh-cn/web/api/virtualkeyboard/show/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## 示例
+
+下面的代码片段展示了如何使用 `virtualkeyboardpolicy` 属性阻止浏览器在单击或点击时显示虚拟键盘。代码还使用 `navigator.virtualKeyboard.show()` 和 `navigator.virtualKeyboard.hide()` 方法，在点击按钮时显示和隐藏虚拟键盘：
+
+```js
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/virtualkeyboard/show/index.md:34]: 同 https://github.com/mdn/translated-content/pull/22043/files#r1660507764

File: files/zh-cn/web/api/virtualkeyboard/show/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{domxref("VirtualKeyboard_API", "VirtualKeyboard API", "", "nocode")}}
+- [使用 VirtualKeyboard API 实现完全控制](https://developer.chrome.com/docs/web-platform/virtual-keyboard/)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/virtualkeyboard/show/index.md:70]: 参见 https://github.com/mdn/translated-content/pull/22039/files#r1660418839

**Review by yin1999 — APPROVED** [review]

**Pull request merged**